src/styledComponents/index.js

```diff
@@ -7,7 +7,7 @@ export const Title = styled.div`
   font-weight: 700;
   margin: 0;
   letter-spacing: 0.7px;
-  line-height: 59px;
```

Comment:
@maisyleigh Can play around with line-height, but it should vary if font-size varies